_sass/articles/article.scss

```diff
@@ -143,7 +144,7 @@
   }
 
   // link
-  a:not(.well-anchor) {
+  a:not(.well-anchor, .share) {
```

Comment:
This breaks the styling for normal `a` tags that don't have classes and are inside `.article` elements.